crates/libs/bindgen/src/types/cpp_struct.rs

```diff
@@ -224,6 +227,16 @@ impl CppStruct {
         tokens
     }
 
+    fn can_derive_default(&self) -> bool {
+        !self.has_explicit_layout()
+            && self.def.fields().all(|field| {
+                !matches!(
+                    field.ty(Some(self)),
+                    Type::ArrayFixed(..) | Type::PtrConst(..) | Type::PtrMut(..)
+                )
+            })
```

Comment:
Not sure if this is easier to read?

```suggestion
            && !self.def.fields().any(|field| {
                matches!(
                    field.ty(Some(self)),
                    Type::ArrayFixed(..) | Type::PtrConst(..) | Type::PtrMut(..)
                )
            })
```

Reply:
Not sure either. 🙃